src/widget.ts

```diff
@@ -77,6 +77,8 @@ export const widget = ((): WidgetHelpers | null => {
       logger.info("Widget API is available");
       const api = new WidgetApi(widgetId, parentOrigin);
       api.requestCapability(MatrixCapabilities.AlwaysOnScreen);
+      api.requestCapability("org.matrix.msc4157.send.delayed_event");
+      api.requestCapability("org.matrix.msc4157.update.delayed_event");
```

Comment:
I would expect these capabilities to be requested by the RoomWidgetClient (like most of our capabilities related to MatrixClient usage are). Also, referring to the capabilities as `MatrixCapabilities.SendDelayedEvent` instead of using a string literal would allow us to stabilize them without that being a breaking change.

Reply:
I was considering this but went for this for simplicity.
Was afraid about the overhead and complexity this might result in for andrews PR.
Maybe it makes most sense that I dont touch it and andrew adds those changes to the already existing future event related PR's